### ISSUE TITLE
R 4.X Compatibility

### ISSUE THREAD

**Issue by Hydr0Ge0Dude** [R-ArcGIS/r-bridge-install#110]
After updating to R 4.X, I cannot install arcgisbinding. Are there plans to make a new release of arcgisbinding for R 4.X? If not, is there a workaround?

**Comment by orhuna** [comment]
@Hydr0Ge0Dude we recently updated `arcgisbinding`, the version for 4.2 support is `v300`. 

Would you please try the snippet below:

```
install.packages("arcgisbinding", repos="https://r.esri.com", type="win.binary")
```

We are working on getting `arcgisbinding v300` ready for R 4.0 and 4.1 but it is available for 4.2.

**Comment by scdub** [comment]
@Hydr0Ge0Dude The released v300 build has support for R 4.x, if you find issues with it let us know. Note that R 4.1 is the final release of R which supports 32-bit platforms, so it won't be possible to use R 4.2 with ArcMap, except via background geoprocessing.
